### PR TITLE
Highlight C++11 functions in search result for cplusplus.com

### PR DESCRIPTION
**Pull request by Spiderpowa**

### CODE REVIEW

File: cppman/main.py [hunk 1]
```diff
@@ -61,6 +61,16 @@ def extract_name(self, data):
         name = re.sub(r'&lt;', r'<', name)
         return name
 
+    def extract_std(self, data):
+        """Extract standard from web page."""
+        std = re.search('<h1[^>]*>(.+?)</h1>', data).group(1)
+        # cplusplus.com
+        if re.search('<span[^>]*cpp11warning[^>]*>', std):
+            std = "c++11"
```
Comment (aitjcize) [inline cppman/main.py]: What about C++14 and above?

Comment (Spiderpowa) [reply cppman/main.py]: cplusplus.com does not have C++14 and above....

Comment (aitjcize) [reply cppman/main.py]: Can you add support for cppreference.com?

File: cppman/main.py [hunk 2]
```diff
@@ -156,6 +146,8 @@ def insert_index(self, table, name, url, std=""):
                 names = [prefix + n for n in names]
 
         for n in names:
+            print('INSERT INTO "%s" (name, url, std) VALUES ("%s", "%s", "%s")' %
```
Comment (aitjcize) [inline cppman/main.py]: remove this.

**Pull request merged**